docs/decisions/0002-combined-cookiecutter-repo.rst

```diff
@@ -0,0 +1,37 @@
+2. Combined Cookiecutter Repo
+=============================
```

Comment:
**Reviewer Note:** This is a placeholder ADR.  Someone can complete it in a separate PR when the docs for this repo are completed.